src/pudl/transform/params/ferc1.py

```diff
@@ -3262,6 +3262,64 @@
             },
         ],
     },
+    "depreciation_amortization_summary_ferc1": {
+        "rename_columns_ferc1": {
+            "dbf": {
+                "columns": {
+                    "respondent_id": "utility_id_ferc1_dbf",
+                    "report_year": "report_year",
+                    "spplmnt_num": "spplmnt_num",
+                    "row_number": "row_number",
+                    "row_seq": "row_seq",
+                    "row_prvlg": "row_prvlg",
+                    "depr_expn": "depreciation_expense",
+                    "depr_asset_retire": "depreciation_expense_asset_retirement",
+                    "limterm_elc_plnt": "amortization_limited_term_electric_plant",
+                    "othr_elc_plnt": "amortization_other_electric_plant",
+                    "total": "depreciation_amortization_total",
+                    "xbrl_factoid": "functional_classification",
```

Comment:
Is `xbrl_factoid` really supposed to be here? I don't think that should show up anywhere in the data until the DBF & XBRL data have been concatenated, and then it's usually being renamed by the `merge_xbrl_metadata` method (which brings in any metadata values we want to put in the output table, like the FERC account numbers.

The structural convention that we've adopted in all of the other tables (which is typically what FERC has been doing as well) is that rows contain accounts, and columns may or may not contain another dimension, if it makes sense, e.g. in the `plant_in_service_ferc1` table we've got starting/ending balances + 4 kinds of deltas.

Many of the accounting tables end up only having a single data column, with one or more categorical columns to guide the aggregation of the one dollar value column. In the case of this table I think we may want to have 2 categorical columns: `functional_classification` and `ferc_account_label` with a single dollar-value column, so one could aggregate across all of the `steam_production_plant` or across all of the `depreciation_expense` values on a per-utility-year basis, without needing to aggregate both across rows and columns. This would also allow the FERC Account metadata (account numbers / description) to be merged onto the table as has been done in other FERC account oriented tables (rather than having the accounts associated with columns)

Reply:
`xbrl_factoid` comes from the merge in `align_row_numbers_dbf`. This gives us the mapping between row numbers and the `functional_classification` categories. `rename_columns` occurs right after the row alignment, so the `xbrl_factoid` column does exist at this point.

Ah ok, I've also added added another layer of stacking using `wide_to_tidy`, that I think satisfies this structural convention.

Reply:
Ah yeah you're right, `xbrl_factoid` is showing up in a lot of the renames. Sorry.